docs/streams/stream-basics/message-queues/index.md

```diff
@@ -0,0 +1,10 @@
+---
+sidebar_position: 1
+title: Message Queues
+---
+
+A message queue automatically balances loads across all consumers for a given stream. Best practice is to use multiple consumers.
```

Comment:
How do I create multiple consumers? Or even one?

Reply:
@diana-macrometa I'll update this to add a link once we've merged the other PR which adds the "Create Consumers" topic